src/workflow-engine/charms/temporal-worker/oci_factory/activities/find_images_to_update.py

```diff
@@ -200,6 +200,11 @@ def find_released_revisions(releases_json: dict) -> dict:
                     release_to[str(to_track)] = {"risks": [to_risk]}
                 else:
                     release_to[to_track]["risks"].append(to_risk)
+                # add end-of-life field to each track
+                if releases[to_track].get("end-of-life"):
```

Comment:
but shouldn't we be checking that the image is still supported? if it has reached its EOL, we shouldn't be updating it.

Similarly for the Continuous-Testing, we should be using the new _release.json fields to skip images that no longer should be scanned